invenio_vocabularies/contrib/affiliations/config.py

```diff
@@ -21,28 +21,27 @@ class AffiliationsSearchOptions(SearchOptions):
     """Search options."""
 
     suggest_parser_cls = SuggestQueryParser.factory(
-        fields=[  # TODO: Tweak with name and acronym
-            'id.text^100',
-            'id.text._2gram',
-            'id.text._3gram',
-            'title.en^5',
-            'title.en._2gram',
-            'title.en._3gram',
+        fields=[
+            'name^100',
+            'acronym^20',
+            'title.*^5',
```

Comment:
In other places we have it as only `en`, i.e. `title.en^5` which will make the suggester only apply to English content... e.g. the [generic vocabulary suggester](https://github.com/inveniosoftware/invenio-vocabularies/blob/31b18084d32c93dbdbc62ff92034fba220a85503/invenio_vocabularies/services/service.py#L41).

If someone wants to test what I mean, you can edit the this by replacing the `*` by `en` and run the test `test_affiliations_suggest_sort` at the bottom. Nothing will be found when `suggest=nucleaire` since it is inside `fr` and not `en`.

I can fix the rest in a separate issue/PR if desired. It would also made a *good first issue*. With this as reference :)